src/Application/Messages/Queries/AllMessageQuery.cs

```diff
@@ -0,0 +1,23 @@
+﻿using CoduTeam.Application.ChatFeature.Models;
+using CoduTeam.Application.ChatFeature.Queries;
+using CoduTeam.Application.Common.Interfaces;
+using CoduTeam.Application.Messages.Models;
+
+namespace CoduTeam.Application.Messages.Queries;
+public class AllMessageQuery : IRequest<IEnumerable<MessageResponse>>
```

Comment:
add Admin authorized attribute 
and make a comment that query only for test purpouse